src/java.net.http/share/classes/module-info.java

```diff
@@ -25,7 +25,125 @@
+ * in which case, any value in {@code conf/net.properties} is overridden. No guarantee is
+ * provided that property values can be set programatically with {@code System.setProperty()}.
+ * Other implementations of this API may choose not to support these properties.
+ * <ul>
+ * <li><p><b>{@systemProperty jdk.httpclient.allowRestrictedHeaders}</b> (default: see below)<br>
+ * A comma-separated list of normally restricted HTTP header names that users may set in HTTP requests
+ * or by user code in HttpRequest instances. By default, the following request headers are not allowed
+ * to be set by user code: connection, content-length, expect, host, and upgrade. You can override this
+ * behavior with this property. The names are case-insensitive and whitespace is ignored. Note that this
+ * property is intended for testing and not for real-world deployments. Protocol errors or other undefined
+ * behavior are likely to occur when using this property. There may be other headers that are restricted
+ * from being set depending on the context. This includes the "Authorization" header when the relevant
+ * HttpClient has an authenticator set. These restrictions cannot be overridden by this property.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.bufsize}</b> (default: 16384 bytes or 16 kB)<br>
+ * The size to use for internal allocated buffers in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.connectionPoolSize}</b> (default: 0)<br>
+ * The maximum number of connections to keep in the HTTP/1.1 keep alive cache. A value of 0 means that
+ * the cache is unbounded.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.connectionWindowSize}</b> (default: 2^26)<br>
+ * The HTTP/2 client connection window size in bytes. The maximum size is 2^31-1. This value cannot be
+ * smaller than the stream window size.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.disableRetryConnect}</b> (default: false)<br>
+ * Whether automatic retry of connection failures is disabled. If false, then retries are attempted
+ * (subject to the retry limit).
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.enableAllMethodRetry}</b> (default: false)<br>
+ * Whether it is permitted to automatically retry non-idempotent HTTP requests.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.enablepush}</b> (default: 1)<br>
+ * Whether HTTP/2 push promise is enabled. A value of 1 enables push promise; a value of 0 disables it.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.hpack.maxheadertablesize}</b> (default: 16384 or 16 kB)<br>
+ * The HTTP/2 client maximum HPACK header table size in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.HttpClient.log}</b> (default: none)<br>
+ * Enables high-level logging of various events through the Java Logging API (which is contained in the
+ * package java.util.logging). The value contains a comma-separated list of any of the following items:
```

Comment:
java.util.logging is actually only the default backend, so I'd suggest to reword as:
```
 * Enables high-level logging of various events through the {@linkplain java.lang.System.Logger Platform Logging API}.
 * The value contains ...
 ```